Microsoft.DotNet.Interactive.Jupyter/ExecuteRequestHandler.cs

```diff
@@ -78,11 +78,11 @@ public async Task Handle(JupyterRequestContext context)
                 default:
                     traceBack.Add("Unhandled Exception");
```

Comment:
Does the output look correct if the traceback is added as a single string with line breaks?